crates/testing/src/script.rs

```diff
@@ -179,6 +179,15 @@ pub struct Expectations<TYPES: NodeType, S> {
     pub task_state_asserts: Vec<Box<dyn Predicate<S>>>,
 }
 
+impl<TYPES: NodeType, S> Expectations<TYPES, S> {
+    pub fn from_outputs(output_asserts: Vec<Box<dyn Predicate<Arc<HotShotEvent<TYPES>>>>>) -> Self {
```

Comment:
@ss-es I added this because we only assert in two tests in the entire codebase, so it was easier than writing the same line over and over, and I don't think it hurts readability, though I could be swayed on a different name.

Reply:
Looks good to me! Honestly, you could probably even just call it `outputs`.